ts/modules/draggable-points.src.ts

```diff
@@ -0,0 +1,3045 @@
+        }
+    }
+);
+
+/**
+ * The draggable-points module allows points to be moved around or modified in
+ * the chart. In addition to the options mentioned under the `dragDrop` API
+ * structure, the module fires three events,
+ * [point.dragStart](plotOptions.series.point.events.dragStart),
+ * [point.drag](plotOptions.series.point.events.drag) and
+ * [point.drop](plotOptions.series.point.events.drop).
+ *
+ * It requires the `modules/draggable-points.js` file to be loaded.
+ *
+ * @sample highcharts/dragdrop/resize-column
+ *         Draggable column and line series
+ * @sample highcharts/dragdrop/bar-series
+ *         Draggable bar
+ * @sample highcharts/dragdrop/drag-bubble
+ *         Draggable bubbles
+ * @sample highcharts/dragdrop/drag-xrange
+ *         Draggable X range series
+ *
+ * @since     6.2.0
+ * @apioption plotOptions.series.dragDrop
+ */
+
+/**
+ * The amount of pixels to drag the pointer before it counts as a drag
+ * operation. This prevents drag/drop to fire when just clicking or selecting
+ * points.
+ *
+ * @type      {number}
+ * @default   2
+ * @since     6.2.0
+ * @apioption plotOptions.series.dragDrop.dragSensitivity
+ *
+ * @private
+ */
+var defaultDragSensitivity = 2;
+
+/**
+ * Style options for the guide box. The guide box has one state by default, the
+ * `default` state.
+ *
+ * @type         {Highcharts.Dictionary<Highcharts.PlotSeriesDragDropGuideBoxDefaultOptions>}
+ * @since        6.2.0
+ * @optionparent plotOptions.series.dragDrop.guideBox
+ *
+ * @private
+ */
+var defaultGuideBoxOptions: (
+    Highcharts.Dictionary<Highcharts.DragDropGuideBoxOptionsObject>
+) = {
+    /**
+     * Style options for the guide box default state.
+     *
+     * @since 6.2.0
+     */
+    'default': {
+        /**
+         * CSS class name of the guide box in this state. Defaults to
+         * `highcharts-drag-box-default`.
+         *
+         * @since 6.2.0
+         */
+        className: 'highcharts-drag-box-default',
+
+        /**
+         * Width of the line around the guide box.
+         *
+         * @since 6.2.0
+         */
+        lineWidth: 1,
+
+        /**
+         * Color of the border around the guide box.
+         *
+         * @type  {Highcharts.ColorString}
+         * @since 6.2.0
+         */
+        lineColor: '#888',
+
+        /**
+         * Guide box fill color.
+         *
+         * @type  {Highcharts.ColorString|Highcharts.GradientColorObject|Highcharts.PatternObject}
+         * @since 6.2.0
+         */
+        color: 'rgba(0, 0, 0, 0.1)',
+
+        /**
+         * Guide box cursor.
+         *
+         * @since 6.2.0
+         */
+        cursor: 'move',
+
+        /**
+         * Guide box zIndex.
+         *
+         * @since 6.2.0
+         */
+        zIndex: 900
+    }
+};
+
+
+/**
+ * Options for the drag handles.
+ *
+ * @since        6.2.0
+ * @optionparent plotOptions.series.dragDrop.dragHandle
+ *
+ * @private
```

Comment:
Same as `defaultGuideBoxOptions` - these should go in the public API - does the `@private` prevent this?

Reply:
The `@private` prevents them to go into the global namespace of the class-reference. `@ignore-option` is the equivalent for the API option tree.